docs/sources/enterprise/auditing.md

```diff
@@ -0,0 +1,166 @@
++++
+title = "Auditing"
+description = "Auditing"
+keywords = ["grafana", "auditing", "audit", "logs"]
+type = "docs"
+[menu.docs]
+parent = "enterprise"
+weight = 700
++++
+
+# Auditing
+
+> Only available in Grafana Enterprise v7.3+.
+
+Auditing allows you to track important changes to your Grafana instance. Modifications to resources such as dashboards and data sources or a user failing to login will all result in an audit log being created. Learn more about what is being audited below.
+By default, audit logs are logged to file but the auditing feature also supports sending logs directly to Loki.
+
+## Audit logs
+
+### Format
+
+Audit logs are JSON objects containing the following fields. The fields followed by **\*** are always available, the others depends on the request made.
```

Comment:
Request made where? What request?

Reply:
Perhaps say `...depends on what part of the Grafana API is being used.`